packages/gatsby/src/utils/worker/render-html.js

```diff
@@ -25,7 +14,12 @@ export function renderHTML({ htmlComponentRendererPath, paths, envVars }) {
         const htmlComponentRenderer = require(htmlComponentRendererPath)
         try {
           htmlComponentRenderer.default(path, (throwAway, htmlString) => {
-            resolve(fs.outputFile(generatePathToOutput(path), htmlString))
+            resolve(
+              fs.outputFile(
+                generateHtmlPathToOutput(`${process.cwd()}/public`, path),
```

Comment:
should not better `path.join` than string concat to be more 1:1 to the previous code?

Reply:
Done 🙇